front/lib/workspace_usage.ts

```diff
@@ -104,6 +150,301 @@ export async function unsafeGetUsageData(
   return csvHeader + csvContent;
 }
 
+export async function unsafeGetMessageUsageData(
```

Comment:
noob question: why unsafe? (maybe a com would help?)

Reply:
No reason tbh, just got the naming pattern of the already existing function. Removing the "unsafe" prefix 👍🏼